src/Log/LogCategory.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Give\Log;
+
+/**
+ * Class LogCategory
+ * @package Give\Log\ValueObjects
+ *
+ * @since 2.9.7
+ */
+class LogCategory {
```

Comment:
If the intention is to make them into Value Objects, then leave them in the directory you had them in. Since these are being made such that other places use the constant, it's important to have a solid namespace. Also, just living in the domain root doesn't give any context. It shouldn't take long to make them into simple VO's.

Reply:
This is not resolved with 067d79ea

I see now that I have an abstract static method. I'll include that logic with an interface.

Reply:
Resolved: 6e95152f d7a286ab